src/browser_action/browser_action.css

```diff
@@ -1,8 +1,160 @@
-.number {
-    font-weight: bold;
-    color: #222222;
+/* General */
+@import url(animate.css);
+@import url(http://fonts.googleapis.com/css?family=Source+Sans+Pro:400,600);
+
+/* Toolbar - General */
+.reviewque-toolbar {
```

Comment:
NOTE: scoping resets to just the toolbar UI (in case w want more UI down the road).